custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -549,4 +548,6 @@ def serialize(person_case):
             for prop in BETSBeneficiaryPayloadGenerator.case_properties
         }
         case_json["properties"]["owner_id"] = person_case.owner_id
+        # This is the "real" phone number
+        case_json["properties"]["phone_number"] = case_properties.get("contact_phone_number", "")
```

Comment:
I added an example in that format to the spec doc, although we didn't discuss specifically.